libs/expo/betterangels/src/lib/screens/Note/NoteTitle.tsx

```diff
@@ -18,8 +18,6 @@ export default function NoteTitle({
       )}
       <TextRegular mb="sm" size="sm">
         {format(new Date(note?.interactedAt), 'MM/dd/yyyy')}
-        {' @ '}
-        {format(new Date(note?.interactedAt), 'hh:mm a')}
```

Comment:
this should be left in. It's removing time from the BA note, which shouldn't be touched